engine/src/main/java/org/terasology/logic/players/LocalPlayerSystem.java

```diff
@@ -489,10 +491,13 @@ private void updateCamera(CharacterMovementComponent charMovementComp, Vector3f
             }
         }
 
-        if (charMovementComp.mode == MovementMode.GHOSTING) {
-            playerCamera.extendFov(24);
-        } else {
-            playerCamera.resetFov();
+        if(charMovementComp.mode != lastMode){
+            if (charMovementComp.mode == MovementMode.GHOSTING) {
```

Comment:
Ahh good ole Doom days :-)

But yeah noclip may make sense if you're a seasoned gamer, but just the term on its own always stuck out to me as weird. 

We have a separate `flight` command that works without the noclip or camera tweaks. Could `ghost` be something else? Prolly. `spectator` I would think of as something else (having no body)